easybuild/easyconfigs/e/ESPResSo/ESPResSo-4.2.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'CMakeMake'
+
+name = 'ESPResSo'
+version = '4.2.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://espressomd.org/wordpress'
+description = """A software package for performing and analyzing scientific Molecular Dynamics simulations."""
+
+source_urls = ['https://github.com/espressomd/espresso/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['e6e17fafa9ba80b46f542dc86e07677a8d641cfd0102ebc4db2f7ee47d5fb6f1']
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+builddependencies = [('CMake', '3.20.1')]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('CUDA', '11.3.1', '', SYSTEM),
+    ('SciPy-bundle', '2021.05'),
+    ('Boost.MPI', '1.76.0'),
+    ('Cython', '0.29.34'),
+    ('HDF5', '1.10.7'),
+    ('Mesa', '21.1.1'),
+    ('GSL', '2.7'),
+    ('IPython', '7.25.0'),
+    ('Pint', '0.20.1'),
+]
+
+configopts = ' -DCMAKE_SKIP_RPATH=OFF -DWITH_TESTS=ON'
+
+separate_build_dir = True
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
+_binaries = ['ipypresso',  'pypresso']
+_libs = [
+    'Espresso_config.so', 'Espresso_core.so', 'Espresso_script_interface.so', 'Espresso_shapes.so',
+    '_init.so', 'analyze.so', 'code_info.so', 'cuda_init.so', 'electrokinetics.so', 'galilei.so',
+    'integrate.so', 'interactions.so', 'lb.so', 'particle_data.so', 'polymer.so', 'profiler.so',
+    'script_interface.so', 'system.so', 'thermostat.so', 'utils.so', 'version.so',
+]
```

Comment:
Could you skip the .so here and append using SHLIB_EXT  in the loop below?

Reply:
thanks, will do

Reply:
done